apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/AbstractServletInstrumentation.java

```diff
@@ -40,6 +42,11 @@
         return Collections.singleton(SERVLET_API);
     }
 
+    @Override
+    public List<String> helpers() throws Exception {
+        return PackageScanner.getClassNames(getClass().getPackage().getName());
```

Comment:
Same- since this would be a reference for future usages, I think it's better to extract the `@RegisterMethodHandle` methods to dedicated classes in dedicated packages. It's easier to make mistakes if we mix instrumentations and advices with those. It also makes sense that packages reflect the complicated class loading contexts.